airflow/ui/.neutrinorc.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
       neutrino.config.resolve.alias.set('utils', resolve(__dirname, 'src/utils'));
       neutrino.config.resolve.alias.set('auth', resolve(__dirname, 'src/auth'));
       neutrino.config.resolve.alias.set('components', resolve(__dirname, 'src/components'));
+      neutrino.config.resolve.alias.set('containers', resolve(__dirname, 'src/containers'));
```

Comment:
Many of the containers are specific to a set of views. Would it not be better for them to live closer to them? (ie `/views/access/Container.tsx` is the AccessContainer.